test/model/src/core/kv_table.cpp

```diff
@@ -271,6 +272,28 @@ kv_table::truncate(kv_transaction_ptr txn, const data_value &start, const data_v
     auto stop_iter = stop == model::NONE ? _data.end() : _data.upper_bound(stop);
 
     try {
+        /* FIXME-WT-13232 Disable this check or make it FLCS only (depending on the fix). */
+        /*
+         * WiredTiger's implementation of truncate returns a prepare conflict if the key following
+         * the truncate range belongs to a prepared transaction, or in some cases, the preceding
```

Comment:
The wording here made this a little hard for me to follow - it sounds like the the key following the truncate range somehow belongs to the preceding key.

How about "if the key following (or, in some cases, preceding) the truncate range belongs to a prepared transaction"?

Reply:
Thanks—that's indeed a better way to put it! Fixed in [484b2cb](https://github.com/wiredtiger/wiredtiger/pull/10933/commits/484b2cbd17ba148c536e6d6494161144c18457f7).